LICENSE

```diff
@@ -1,22 +1,106 @@
-MIT License
+Business Source License 1.1
 
-Copyright (c) 2020, MIT Data To AI Lab
+Parameters
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+Licensor:             DataCebo, Inc.
 
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
+Licensed Work:        DeepEcho
+                      The Licensed Work is (c) DataCebo, Inc.
 
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+Additional Use Grant: You may make use of the Licensed Work, and derivatives of the Licensed
+                      Work, provided that you do not use the Licensed Work, or derivatives of
+                      the Licensed Work, for a Synthetic Data Creation Service.
 
+                      A "Synthetic Data Creation Service" is a commercial offering
+                      that allows third parties (other than your employees and
+                      contractors) to access the functionality of the Licensed
+                      Work so that such third parties directly benefit from the
+                      data processing, machine learning or synthetic data creation
+                      features of the Licensed Work.
+
+Change Date:          Change date is four years from release date.
+Please see https://github.com/sdv-dev/DeepEcho/releases for exact dates.
```

Comment:
Let's put some spaces at the beginning of this line. It the "Please" should start right underneath the sentence "Change date is four years ..."